GVRf/Framework/src/org/gearvrf/GVRMesh.java

```diff
@@ -17,20 +17,26 @@
 
 import static org.gearvrf.utility.Assert.*;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.gearvrf.utility.Exceptions;
+import org.gearvrf.utility.Log;
 
 /**
  * This is one of the key GVRF classes: It holds GL meshes.
  * 
  * A GL mesh is a net of triangles that define an object's surface geometry.
  */
-public class GVRMesh extends GVRHybridObject {
+public class GVRMesh extends GVRHybridObject implements PrettyPrint {
```

Comment:
it's intended to be a developer's helper to view internal data structures of so-called core classes. framework developers can use them to understand how importers create those structures and quickly detect issues.